.github/workflows/remote-build-android-hybrid.yml

```
@@ -0,0 +1,30 @@
+name: Remote Build Android Hybrid
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
```

Let's add a concurrency key (for all these workflows) to save on runner minutes.

```suggestion
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
```

I chose not to include `cancel-in-progress: true`, because I feel like sometimes we have frequent enough pushes to main that we might go the whole workday without builds finishing.